src/types/index.ts

```diff
@@ -20,6 +20,14 @@ interface Files {
   [index: string]: string;
 }
 
+/**
+ * The domain of a repository
+ */
+interface RepoDomain {
+  repo: string;
+  owner: string;
+}
```

Comment:
Another conceptual model that we may want to consider is repo+owner+branch name. I think we could possibly have another interface:
```ts

interface RepoBranch extends RepoDomain {
  branch: string;
}
```

Reply:
Let's table this discussion for when we want to support multiple upstream origin repositories